source/Ship.cpp

```diff
@@ -1120,8 +1120,10 @@ vector<string> Ship::FlightCheck() const
 			checks.emplace_back("afterburner only?");
 		if(!thrust && !afterburner)
 			checks.emplace_back("reverse only?");
-		if(!generation && !solar && !consuming)
+		if(!generation && !solar && !consuming && !canBeCarried)
```

Comment:
Why not show the warning for carried ships? It's still risky to use them, no more than non carried ships. No?